src/mongocxx/options/create_view.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <mongocxx/options/create_view.hpp>
+
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/types.hpp>
+
+#include <mongocxx/config/private/prelude.hpp>
+
+namespace mongocxx {
+MONGOCXX_INLINE_NAMESPACE_BEGIN
+namespace options {
+
+create_view& create_view::pipeline(class pipeline pipeline) {
+    _pipeline = std::move(pipeline);
+    return *this;
+}
+
+bsoncxx::document::value create_view::to_document() const {
```

Comment:
Let's leave this as is and expand CXX-1053, but let's change that to 3.1-required.  That way we can close this PR/ticket but won't lose track.  Unlike the server, we have much more direct control of what we ship versus what we drop (as you've seen as we've re-triaged the list a couple times).